apps/fz_wall/lib/fz_wall/cli/live.ex

```diff
@@ -42,16 +42,41 @@ defmodule FzWall.CLI.Live do
         "{ type nat hook postrouting priority 100 ; }'"
     )
 
-    # XXX: Do more testing with this method of creating masquerade rules
-    for int <- File.ls!("/sys/class/net/") do
-      # Masquerade all interfaces except loopback and our own wireguard interface
-      if int not in ["lo", wireguard_interface_name()] do
+    if masquerade_ipv4?() || masquerade_ipv6?() do
```

Comment:
can we just call `setup_masquerade` since that already test for the configuration to add the corresponding rules?

Reply:
Hmm we could, but the File.ls! call and following Enum functions would run needlessly. I’ll see if I can clean it so I don’t check for the conditional twice.

Reply:
Updated